### PR TITLE
Update login signup

### PR DESCRIPTION
**Pull request by Sahil-Khowaja**

### CODE REVIEW

**Comment by sahilkhowaja** [comment]
image
image

**Comment by salman299** [comment]
Good Work! 
Here are the suggestions:
Update the file name from "new_logo.png" to the "logo-icon.png", it's more meaningful than the "new_logo.png"

File: src/ui-component/AuthLogo.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  return (
+    <img src={logo} alt="Community Verse" width="200" />
+    /**
+     * if you want to use image instead of svg uncomment following, and comment out <svg> element.
+     *
+     * <img src={logo} alt="Berry" width="100" />
+     *
+     */
+    
+    // <svg width="92" height="32" viewBox="0 0 92 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (salman299) [inline src/ui-component/AuthLogo.js]: Remove extra code from this function

File: src/ui-component/AuthLogo.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ * if you want to use image instead of <svg> uncomment following.
+ *
+ * import logoDark from 'assets/images/logo-dark.svg';
+ * import logo from 'assets/images/logo.svg';
+ *
+ */
+
+// ==============================|| LOGO SVG ||============================== //
+
+const Logo = () => {
```
Comment (salman299) [inline src/ui-component/AuthLogo.js]: Update this function to AuthLogo

File: src/ui-component/AuthLogo.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    //   />
+    //   <path
+    //     d="M21.2765 4.32541C21.5343 3.21728 21.6681 1.90776 21.6881 0.41748C15.9226 1.70883 13.3224 4.17658 15.2839 8.33846C15.3816 8.36203 15.4754 8.38119 15.5696 8.40085C16.0281 5.14422 18.0463 3.93835 21.2765 4.32541Z"
+    //     fill={theme.palette.primary.main}
+    //   />
+    // </svg>
+  );
+};
+
+export default Logo;
```
Comment (salman299) [inline src/ui-component/AuthLogo.js]: Also update this to AuthLogo

File: src/ui-component/AuthLogo.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
+// material-ui
+// import { useTheme } from '@mui/material/styles';
+import logo from 'assets/images/new-logo.svg';
```
Comment (salman299) [inline src/ui-component/AuthLogo.js]: Update the name of the logo to logo-icon.png

**Comment by salman299** [comment]
Also, remove the yarn.lock file from the commits.

**Review by salman299 — APPROVED** [review]

**Pull request merged**